modules/gapi/test/common/gapi_video_tests_inl.hpp

```diff
@@ -131,6 +131,272 @@ TEST_P(BackgroundSubtractorTest, AccuracyTest)
     // Allowing 1% difference of all pixels between G-API and reference OpenCV results
     testBackgroundSubtractorStreaming(gapiBackSub, pOCVBackSub, 1, 1, learningRate, testNumFrames);
 }
+
+TEST_P(KalmanFilterTest, AccuracyTest)
+{
+    cv::gapi::video::KalmanParams kp;
+    kp.type = type;
+
+    kp.statePre = Mat::zeros(dDim, 1, type);
+    kp.errorCovPre = Mat::zeros(dDim, dDim, type);
+    kp.statePost = Mat::zeros(dDim, 1, type);
+    kp.errorCovPost = Mat::zeros(dDim, dDim, type);
+
+    kp.transitionMatrix = Mat::ones(dDim, dDim, type)*2;
+    kp.processNoiseCov = Mat::eye(dDim, dDim, type);
+    kp.measurementMatrix = Mat::eye(mDim, dDim, type)*2;
+    kp.measurementNoiseCov = Mat::eye(mDim, mDim, type);
+
+    if (cDim > 0)
+        kp.controlMatrix = Mat::eye(dDim, cDim, type);
+
+    cv::randu(kp.statePre, Scalar::all(-1), Scalar::all(1));
+
+    setIdentity(kp.measurementMatrix);
+    setIdentity(kp.processNoiseCov, Scalar::all(1e-5));
+    setIdentity(kp.measurementNoiseCov, Scalar::all(1e-1));
+    setIdentity(kp.errorCovPre, Scalar::all(1e-5));
```

Comment:
And please specify `cv::` namespace for `setIdentity` and all other `cv::` functions

Reply:
outdated